consensus/src/experimental/buffer_manager.rs

```diff
@@ -254,6 +184,112 @@ impl StateManager {
         Ok(())
     }
 
+    /// this function updates the buffer according to the response from the execution phase
+    /// it also initiates a request to the signing phase.
+    async fn update_buffer_execution_resp(
```

Comment:
to keep the naming consistent, this is more like process_successful_execution_response and only take the Ok() result as params